maps/graph_edges/base_graph/edges_dgb.py

```diff
@@ -56,8 +56,7 @@
     {"from": {"map": "DGB_03", "id": 4}, "to": {"map": "DGB_03", "id": 5}, "reqs": []}, #? Table/Clock Room (1/2F) Door 2F Far Right -> Table/Clock Room (1/2F) 2F Behind The Clock
     {"from": {"map": "DGB_03", "id": 5}, "to": {"map": "DGB_03", "id": 4}, "reqs": []}, #? Table/Clock Room (1/2F) 2F Behind The Clock -> Table/Clock Room (1/2F) Door 2F Far Right
 
-    {"from": {"map": "DGB_03", "id": 4},       "to": {"map": "DGB_03", "id": "ItemA"}, "reqs": []}, #* Table/Clock Room (1/2F) Door 2F Far Right -> ItemA (StarPiece)
-    {"from": {"map": "DGB_03", "id": "ItemA"}, "to": {"map": "DGB_03", "id": 0},       "reqs": []}, #* ItemA (StarPiece) -> Table/Clock Room (1/2F) Door 1F Far Left
```

Comment:
Can this really be removed? Doesn't it remove the only link between the top and bottom exits of the room?

Reply:
Good catch, yup you're correct, this edge shouldn't be removed